storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/impl/DefaultStorIOSQLite.java

```diff
@@ -276,16 +284,18 @@ public int delete(@NonNull DeleteQuery deleteQuery) {
         public void notifyAboutChanges(@NonNull Changes changes) {
             // Notifying about changes requires RxJava, if RxJava is not available -> skip notification
             if (changesBus != null) {
-                changesBus.onNext(changes);
+                pendingChanges.add(changes);
+                notifyAboutPendingChangesIfNotInTransaction();
             }
         }
 
-        /**
-         * {@inheritDoc}
-         */
-        @Override
-        public boolean transactionsSupported() {
-            return true;
+        private void notifyAboutPendingChangesIfNotInTransaction() {
+            if (changesBus != null && numberOfRunningTransactions.get() == 0) {
+                for (final Changes changes : pendingChanges) {
+                    pendingChanges.remove(changes);
```

Comment:
Nope, but I found a problem in this code:

Thread1 executes `notifyAboutPendingChanges()` and sees `numberOfRunningTransactions == 0`, Thread1 starts notifying about changes.

Thread2 executes `notifyAboutPendingChanges()` and sees `numberOfRunningTransactions == 0`, Thread2 starts notifying about changes

Problem: Thread1 and Thread2 can notify about same changes, that's bad.